sdk/core/core-amqp/package.json

```diff
@@ -52,8 +52,8 @@
     "url": "http://github.com/Azure/azure-sdk-for-js/issues"
   },
   "dependencies": {
-    "@azure/abort-controller": "1.0.0-preview.1",
-    "@azure/core-auth": "^1.0.0-preview.1",
+    "@azure/abort-controller": "1.0.0-preview.2",
```

Comment:
This assumes that we have changes in abort-controller to ship. As of now, there are no changes in that package. 

While rush wont complain, this will be a problem when we ship core-amqp or any other package in this PR for which we have now updated the abort-controller dependency